packages/portal/src/lang/en.js

```diff
@@ -984,10 +984,13 @@ export default {
   "pages": {
     "collections": {
       "organisations": {
+        "description": "Explore the institutions which share digital cultural heritage items on Europeana.eu.",
         "table": {
-          "name": "Name"
+          "country": "Country",
+          "name": "Name",
+          "showMoreData": "Show more data for {organisation}"
```

Comment:
This may be reused in other entity types in future, and does not contain any org-specific text, so move to the shared `table` group, and replace `{organisation}` w/ `{entity}`.

Reply:
Moved.